### ISSUE TITLE
Add replace_hashtags, remove_hasthags

### ISSUE THREAD

**Issue by jbesomi** [jbesomi/texthero#30]
(Edit)

Add `replace_hashtags`, `remove_hastags`

**Example**:

```
>>> s = pd.Series(["Hey #git123 how are you doing?"])
>>> hero.replace_tag(s, symbol="X")
0    Hey Xgit123 how are you doing? 
dtype: object
```

**Implementation**

Please refer to the other replace_*, remove_* examples by looking at the code in `preprocessing.py`. `remove_hashtags` should simply call `replace_hashtags` with symbol=` `.

Thank you  [/u/penatbater](https://www.reddit.com/user/penatbater) from Reddit for the suggestion.

**Comment by VinayChaudhari1996** [comment]
Updated 2 Functions,

```
1. Remove @ tags ---> hero.remove_tags()
2. Remove # tags --->  hero.remove_hashtags()
```

**Comment by ishanarora04** [comment]
Hey, this is exactly similar to tags ?

**Comment by jbesomi** [comment]
Yes! :) want to take it? (probably you are the better match out there :))

**Comment by ishanarora04** [comment]
yes, I will pick this up tonite.

**Comment by jbesomi** [comment]
Implemented in #58
